crates/editor/src/signature_help.rs

```diff
@@ -291,28 +327,171 @@ impl SignatureHelpState {
 }
 
 #[derive(Clone, Debug, PartialEq)]
+pub struct SignatureHelpData {
```

Comment:
```suggestion
pub struct SignatureHelp {
```

Just as a naming proposal.